pkg/asset/installconfig/gcp/client.go

```diff
@@ -233,6 +222,35 @@ func (c *Client) GetProjects(ctx context.Context) (map[string]string, error) {
 	return projects, nil
 }
 
+// GetZones uses the GCP Compute Service API to get a list of machine types from a project.
+func (c *Client) GetZones(ctx context.Context, project, filter string) (map[string]*compute.Zone, error) {
```

Comment:
I think we should just return a slice of zones rather than going through the effort of making a map. This makes it easier on the only current caller, too.
```suggestion
func (c *Client) GetZones(ctx context.Context, project, filter string) ([]*compute.Zone, error) {
```

Reply:
Done: https://github.com/openshift/installer/commit/60a57050d1738c13289e883ac5f32cf338d2ea0f